src/mlpack/methods/ann/layer/dropout_impl.hpp

```diff
@@ -77,18 +77,22 @@ DropoutType<MatType>::operator=(DropoutType&& other)
 template<typename MatType>
 void DropoutType<MatType>::Forward(const MatType& input, MatType& output)
 {
-  // The dropout mask will not be multiplied in testing mode.
   if (!this->training)
   {
     output = input;
   }
   else
   {
-    // Scale with input / (1 - ratio) and set values to zero with probability
-    // 'ratio'.
```

Comment:
@MarkFischinger do you think you can re-add these comments?  I think they are useful to keep.